### ISSUE TITLE
Text scale in segments

### ISSUE THREAD

**Issue by ogdenwebb** [dbordak/telephone-line#79]
Is there built-in ability to scale text within telephone-line? Would you add something similar to powerline-text-scale-factor?  Probably this related to a generic padding keyword in #40.

**Comment by dbordak** [comment]
If you're talking about the whole bar, why not just change the font size? The separator height changes automatically (although you might have to clear the cache).

Per-segment is kinda dicey and I don't think it'd work.

**Comment by ogdenwebb** [comment]
> If you're talking about the whole bar, why not just change the font size?

Yes, generally I ask about change the font size whole mode-line. I've tried, but my mode-line doesn't fit well. 

![2018-08-20-140446_962x69_scrot](https://user-images.githubusercontent.com/9018005/44337133-382d5980-a482-11e8-8908-7c286676fcc5.png)


I tried this. 
```emacs-lisp  
(set-face-attribute 'mode-line nil :font "DejaVu Sans Mono-10")
```
Even if I set `:height 100` for example in face definition, my mode-line still looks like the above.

> although you might have to clear the cache

How can I achieve that?

**Comment by dbordak** [comment]
Sorry, this isn't what I thought it was at first -- this is just a bug. I'll look into it next week, don't have any time for it right now.

**Comment by ogdenwebb** [comment]
> can't really see from the screenshot?
No, I don't use them.

> Alright, just in case, here's how you clear the caches
It doesn't help though, the same result as above.

**Comment by sebasmonia** [comment]
@ogdenwebb it's likely that the (partial) merge in PR #104 will help with this issue, FYI.
